components/main-chef-wrapper/cmd/root.go

```diff
@@ -93,11 +94,11 @@ func init() {
 	// These flags are common to all child commands.  Some of them do not need config or debug,
 	// so we can look at pushing this down; but it seems to make sense since it's present for more
 	// commands than it isn't.
-	rootCmd.PersistentFlags().StringVarP(&options.configFile, "config", "c", "", "Read configuration from `CONFIG_FILE_PATH`")
-	rootCmd.PersistentFlags().StringVar(&options.licenseAcceptance, "chef-license", "",
-		"Accept product license, where `ACCEPTANCE` is one of 'accept', 'accept-no-persist', or 'accept-silent'")
-	rootCmd.PersistentFlags().BoolVarP(&options.debug, "debug", "d", false,
-		"Enable debug output when available")
+	// rootCmd.PersistentFlags().StringVarP(&options.configFile, "config", "c", "", "Read configuration from `CONFIG_FILE_PATH`")
```

Comment:
I am not clear what the changes are between commented/uncommented versions; but in any case we should remove the commented-out code.

Reply:
I removed this because these flags are global here but  these flags are not applicable to many commands in chef-cli, but anyhow adding these flags again here as this is a bug in chef-cli and these are the high level flags should be applicable to all the commands in chef-cli. I raised an issue for the same and will pick after this.